spec/tic_tac_toe/ui_spec.clj

```diff
@@ -0,0 +1,57 @@
+(ns tic-tac-toe.ui-spec
+  (:require [speclj.core :refer :all]
+            [tic-tac-toe.ui :refer :all]
+            [tic-tac-toe.ui-console :as ui-console]
+            [tic-tac-toe.board :refer :all]))
+
+(def test-board
+  [X space O
+   O space O
+   X space X])
+
+(def empty-board-3x3
+  [space space space
+   space space space
+   space space space])
+
+(def empty-board-4x4
+  [space space space space
+   space space space space
+   space space space space
+   space space space space])
+
+(defn test-prompt [message]
+  (read-line))
+
+(defn mock-output [message]
+  (do message))
```

Comment:
I believe you would use `do` if you're evaluating multiple expressions. Here you can just return `message`.